cabal-install/tests/UnitTests/Distribution/Solver/Modular/Solver.hs

```diff
@@ -181,6 +182,9 @@ tests =
       , runTest $ mkTest db9 "setupDeps7" ["F", "G"] (solverSuccess [("A", 1), ("B", 1), ("B", 2), ("C", 1), ("D", 1), ("E", 1), ("E", 2), ("F", 1), ("G", 1)])
       , runTest $ mkTest db10 "setupDeps8" ["C"] (solverSuccess [("C", 1)])
       , runTest $ indep $ mkTest dbSetupDeps "setupDeps9" ["A", "B"] (solverSuccess [("A", 1), ("B", 1), ("C", 1), ("D", 1), ("D", 2)])
+      , runTest $ setupStanzaTest1
+      , runTest $ setupStanzaTest2
+      , runTest $ setupStanzaTest3
```

Comment:
There seems to be a conflict between the [wiki guidelines](https://wiki.haskell.org/Programming_guidelines#Naming_Conventions) and the local file convention

Reply:
@aleeusgr: would you mind being more descriptive? what conflict precisely? these are no longer notes to self, so that you can explain your point if asked, but a form of communication, which requires some extra effort and verbosity

@mpickering: IIRC, this is one of the first cabal PRs reviewed by @aleeusgr ever, who kindly responded to my plea for more reviews, so let's please guide @aleeusgr in this endeavour. :)

Reply:
The wiki says names should be descriptive, but here is an article that suggests that refactoring tests and giving them names isn't something worth doing.

What is the return on intention of such a proposal? Is it worth going through the whole file and naming tests in a description fashion rather than `test1`, `test2`, etc?